.gitignore

```diff
@@ -3,4 +3,7 @@ coverage
 node_modules
 
 # Pika builds
-pkg/
\ No newline at end of file
+pkg/
+
+# TypeScript generated compiled scripts
+scripts/*.js
```

Comment:
Would you mind to leave the `scripts/` files written in JavaScript please? Having them in TypeScript is causing unnecessary maintenance overhead / complexity.

Reply:
> Would you mind to leave the `scripts/` files written in JavaScript please? Having them in TypeScript is causing unnecessary maintenance overhead / complexity.

Yes! No problem! It will simplify a lot the comments we have in place right now and... if we came up in the future a way to have it in typescript in a simple way we are always on time for that migration :)